bin/smack.py

```diff
@@ -376,6 +401,8 @@ def verify_bpl(args):
     command += ["/cex:%s" % args.max_violations]
     command += ["/maxStaticLoopBound:%d" % args.loop_limit]
     command += ["/recursionBound:%d" % args.unroll]
+    if args.context_switches:
```

Comment:
Monty, this part is wrong. If you look several lines above this, you will see that we are already passing /k:1 to Corral. The reason I had to do this is because Corral by default in fact sets k to 2. So your context_switches parameter should actually be set to 1 by default, and then slightly rewrite this part so that k is always set, meaning that you don't need the ``if args.context_switches`` part.